src/blockchain/new-block-emitters.ts

```diff
@@ -127,3 +130,21 @@ export class ListeningNewBlockEmitter extends AutoStartStopEventEmitter {
     this.subscription?.unsubscribe(error => { this.logger.error(error) })
   }
 }
+
+export class NewBlockEmitterService implements Partial<ServiceMethods<any>> {
+  emit?: Function
+  events: string[] = []
+
+  constructor () {
+    this.events = [NEW_BLOCK_EVENT_NAME]
+  }
+
+  // eslint-disable-next-line require-await
+  async find (): Promise<BlockHeader> {
+    return this.getLastBlockNumber()
+  }
+
+  getLastBlockNumber (): BlockHeader {
```

Comment:
Small nitpick 😇 It is not "last block **number**" anymore. Can you please rename it?

Reply:
Oops